File: internal/content/testdata/bench/csharp/nc-min-cost-climbing.cs
public class Solution {
    public long min_cost_climbing_stairs(long[] cost) {
        int n = cost.Length;
        for (int i = 2; i < n; i++) {
            cost[i] += Math.Min(cost[i - 1], cost[i - 2]);
        }
        return Math.Min(cost[n - 1], cost[n - 2]);
    }
}

File: internal/content/testdata/bench/typescript/nc-min-cost-climbing.ts
function min_cost_climbing_stairs(cost: number[]): number {
    const n = cost.length;
    const c = cost.slice();
    for (let i = 2; i < n; i++) {
        c[i] += Math.min(c[i - 1], c[i - 2]);
    }
    return Math.min(c[n - 1], c[n - 2]);
}
